frontend/simple/js/distribution/mincome-proportional.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+function incomeDistribution (incomes, minCome) {
+  let membersBelow = []
+  let belowMincomeTotalAmount = 0
+  let membersAbove = []
+  let aboveMincomeTotalAmount = 0
+
+  incomes.map(function (member) {
+    let belowMincome = minCome - member.amount
+    if (belowMincome > 0) {
+      membersBelow.push(member.name)
+      belowMincomeTotalAmount += belowMincome
+    }
+    let aboveMincome = member.amount - minCome
+    if (aboveMincome > 0) {
+      membersAbove.push(member.name)
+      aboveMincomeTotalAmount += aboveMincome
+    }
+  })
+
+  let totalProportionalDistribution = belowMincomeTotalAmount / aboveMincomeTotalAmount
```

Comment:
Doesn't seem like you handled the case where this value is > 1? Remember, in that case you just need to distribute 100% of the `aboveMincomeTotalAmount`